src/commands/Minion/k.ts

```diff
@@ -131,7 +131,7 @@ export default class extends BotCommand {
 
 		// Check requirements
 		const [hasReqs, reason] = msg.author.hasMonsterRequirements(monster);
-		if (!hasReqs) throw reason;
+		if (!hasReqs) return msg.channel.send(`${reason}`);
```

Comment:
If anything related to a continuation trip or new slayer task ended up here, it would result in no message at all. Removed the throw and added a return msg to be more in line to what we have been doing lately.